components/bio-formats-plugins/src/loci/plugins/About.java

```diff
@@ -44,11 +44,11 @@
 
   /** URL of the Bio-Formats web page. */
   public static final String URL_SOFTWARE =
-    "http://www.openmicroscopy.org/site/products/bio-formats";
+    "http://openmicroscopy.org/info/bio-formats";
 
   /** URL of Bio-Formats ImageJ web page. */
   public static final String URL_BIO_FORMATS_IMAGEJ =
-    "http://www.openmicroscopy.org/site/support/bio-formats/users/imagej/features.html";
+    "http://openmicroscopy.org/info/bio-formats/users/imagej/features.html";
```

Comment:
This is not how the info redirects were meant to work. The initial use was `http://openmicroscopy.org/info/bio-formats` and that plus anything below that should redirect to `http://www.openmicroscopy.org/site/products/bio-formats`.

This means that `http://openmicroscopy.org/info/bio-formats/any-sub-path` is not supported and is probably only working by luck.